### ISSUE TITLE
No search results using docker - network issue? Ports are open

### ISSUE THREAD

**Issue by mminklet** [slskd/slskd#897]
Hi, set up slskd on a rpi3 with docker. I am logged in and connected to the network, but cannot get any results. No filters, using obvious searches. I can't browse a user, tho I can search for them, see some basic details (but 'No User Info') and search for chat rooms. I suspect it's due to ports but they are open

I've got a static IP (and soulseek works fine on my mac). I've forwarded the listening port on docker-proxy (50300) and tested it online, it's open to the public.

`tcp        0      0 0.0.0.0:50300           0.0.0.0:*               LISTEN      91794/docker-proxy`

Is there anything else that needs to be done? I have shared a folder, tho there's nothing in it yet

I enabled debug mode but it's not showing me anything in the logs.

This is my info config

```
version:
  full: 0.17.5.0 (0.17.5.0+ea52f17d)
  current: 0.17.5.0
  latest: 0.17.5.0
  isUpdateAvailable: false
  isCanary: false
  isDevelopment: false
pendingReconnect: false
pendingRestart: false
server:
  address: vps.slsknet.org
  ipEndPoint: 208.76.170.59:2271
  state: Connected, LoggedIn
  isConnected: true
  isLoggedIn: true
  isTransitioning: false
relay:
  mode: Controller
  controller:
    address: null
    state: Disconnected
  agents: []
user:
  username: foobar <- not real
  privileges:
    isPrivileged: true
    privilegesRemaining: 1772850
  statistics:
    averageSpeed: 0
    directoryCount: 0
    fileCount: 0
    uploadCount: 0
distributedNetwork:
  branchLevel: 4
  branchRoot: indadb
  canAcceptChildren: true
  childLimit: 25
  children: []
  hasParent: true
  isBranchRoot: false
  parent: Andres17
shares:
  scanPending: false
  scanning: false
  ready: true
  faulted: false
  cancelled: false
  scanProgress: 1
  hosts:
    - local
  directories: 0
  files: 0
rooms: []
users:
  - username: null
    group: null
    statistics:
      averageSpeed: 0
      directoryCount: 0
      fileCount: 0
      uploadCount: 0
    status: null
```

**Comment by jpdillingham** [comment]
I see you've connected to a parent so it's capable of establishing outbound connections at the very least.

Try setting [this option](https://github.com/slskd/slskd/blob/master/config/slskd.example.yml#L126) to `Debug`, restart, then do a search.  You should get a ton of logs that will tell you if you are receiving search results from other clients.  It will probably perform poorly due to the number of logs, just give it time.

Do you see any `.db` files in the application directory? I'm thinking there's a chance it may not be able to write the results to the database.  It should be throwing a bunch of errors but the search stuff is using older (crappier) code.

**Comment by jpdillingham** [comment]
It looks like you can receive incoming connections, so you should be good.  The term 'beatles' is one of a handful that are blocked at the server; it won't forward the search to the rest of the network so you'll never get any responses.  Try searching for 'mp3', you should get a ton of results.

**Comment by mminklet** [comment]
Oh fair enough, I actually chose that because I figured there'd be loads. I was originally searching 'overmono' and also getting no results.

I tried mp3 and it found 26266, and alicia and it found 3000, so it's clearly working. However for 'overmono' or 'overmono Is U' I get nothing

overmono isn't really obscure but obviously more obscure than beatles, but then I get results for other similar sized artists. Is that a blocked term?

is overmono blocked?
